.cbmc-batch/stubs/memmove_override_havoc.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#undef memmove
+
+#include <proof_helpers/nondet.h>
+#include <stdint.h>
+
+/**
+ * Override the version of memmove used by CBMC.
+ */
+void *memmove_impl(void *dest, const void *src, size_t n) {
```

Comment:
Please add a comment explaining _why_ you are overriding it so that future readers of this code understand whether that's still necessary.